locals.pkr.hcl

```diff
@@ -60,4 +60,6 @@ locals {
         ]
     ])
   )
+
+  jdks_infos = yamldecode(file("./jdks-infos.yaml"))
```

Comment:
```suggestion
```

=> conflict resolution issue? The same instruction is already present on line 35